src/S3Policy.js

```diff
@@ -34,8 +34,10 @@ export class S3Policy {
   }
 }
 
-const getDate = () => {
-  let date = new Date();
+const getDate = (options) => {
```

Comment:
As a minor nitpick, do you think instead of passing in `options` here we could just pass in the `timeDelta`?

```javascript
const getDate = (timeDelta) => {
  let date = new Date(
    (new Date).getTime() + (timeDelta || 0)
  );
  // ...
}
```